<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditarCita</class>
 <widget class="QMainWindow" name="EditarCita">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1100</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1150</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Editar cita</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>49</x>
      <y>40</y>
      <width>1051</width>
      <height>641</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTableWidget" name="tablaCitas">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>390</y>
       <width>781</width>
       <height>241</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
*{
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
	font: 9pt &quot;MS Shell Dlg 2&quot;;
}</string>
     </property>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <column>
      <property name="text">
       <string>ID Cita</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ID Paciente</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nombre</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Apellido Paterno</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Apellido Materno</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Fecha de cita</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>10</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Editar Citas</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>80</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>ID de cita</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>150</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>ID del paciente</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>220</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Fecha de cita</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="InputIdCita">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>80</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="inputIdPaciente">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>140</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QComboBox" name="inputA">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>220</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
font: 16pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);</string>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2021</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2022</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2023</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2024</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2025</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2026</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2027</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2028</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2029</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2030</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>220</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>AÃ±o</string>
     </property>
    </widget>
    <widget class="QComboBox" name="inputM">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>220</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>-01-</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>-02-</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>-03-</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>-04-</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>-05-</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>-06-</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>-07-</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>-08-</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>-09-</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>-10-</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>-11-</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>-12-</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>220</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Mes</string>
     </property>
    </widget>
    <widget class="QComboBox" name="inputD">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>220</y>
       <width>73</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>01</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>02</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>03</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>04</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>05</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>06</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>07</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>08</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>09</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>11</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>12</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>13</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>14</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>15</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>16</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>17</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>18</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>20</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>21</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>22</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>23</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>24</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>25</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>26</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>27</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>28</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>29</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>30</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>31</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>220</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Dia</string>
     </property>
    </widget>
    <widget class="QComboBox" name="inputH">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>280</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>00</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>01</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>02</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>03</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>04</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>05</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>06</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>07</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>08</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>09</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>11</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>12</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>13</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>14</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>15</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>16</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>17</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>18</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>20</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>21</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>22</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>23</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="inputMin">
     <property name="geometry">
      <rect>
       <x>502</x>
       <y>280</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:00</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:01</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:02</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:03</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:04</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:05</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:06</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:07</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:08</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:09</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:11</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:12</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:13</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:14</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:15</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:16</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:17</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:18</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:19</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:20</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:21</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:22</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:23</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:24</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:25</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:26</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:27</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:28</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:29</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:30</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:31</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:32</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:33</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:34</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:35</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:36</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:37</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:38</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:39</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:40</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:41</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:42</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:43</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:44</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:45</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:46</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:47</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:48</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:49</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:50</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:51</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:52</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:53</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:54</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:55</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:56</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:57</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:58</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>:59</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>280</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Hora</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>280</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Minutos</string>
     </property>
    </widget>
    <widget class="QPushButton" name="editarCita">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>220</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Editar Cita</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buscarCita">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>80</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Buscar Cita</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1150</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMen">
    <property name="title">
     <string>MenÃº</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionRegresar"/>
   </widget>
   <addaction name="menuMen"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionRegresar">
   <property name="text">
    <string>Regresar</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
